### PR TITLE
Maven prereq >= 3.1.0, added slf4j-simple to resolve missing StaticLoggerBinder error

### PR DESCRIPTION
**Pull request by liebowitz**
Hi, nice plugin. The mojo appears to be compatible with the latest Maven version -- It doesn't appear necessary to restrict it to 3.1.0.  I also added a dependency on slf4j-simple to resolve the missing StaticLoggerBinder error.

### CODE REVIEW

**Comment by abner** [comment]
+1 I was used to use the plugin with maven 3.3
I had to downgrade to older version of plugin because of this added restriction

**Comment by eirslett** [comment]
@stephenc Will including slf4j-simple in the plugin still work, or will we hit the previous class loader issues?

File: frontend-maven-plugin/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <name>Maven Frontend Plugin</name>
     <prerequisites>
-        <maven>3.1.0</maven>
+        <maven>>=3.1.0</maven>
```
Comment (stephenc) [inline frontend-maven-plugin/pom.xml]: that's not a valid range syntax. you probably mean `[3.1.0,)`

Comment (liebowitz) [inline frontend-maven-plugin/pom.xml]: Thanks.  Probably best to revert this change.
